src/ToolingNuget/Nuget/Microsoft.Windows.VbsEnclave.CodeGenerator.targets

```diff
@@ -87,6 +88,7 @@
         Condition="!Exists('$(NugetSdkRoot)')">
         <VBS_Enclave_Dependencies Condition="'$(VBS_Enclave_Dependencies)' == ''">
             vertdll.lib;
+            veinterop.lib
```

Comment:
>veinterop.lib [](http://example.com/codeflow?start=12&length=13)

Does this belong in codegen targets?  Why not SDK targets?

Reply:
100.

(layering is confusing with all the pieces, but will be more confusing if dependencies are just shoved into the lowest common denominator, imo)